commands/provider/workflow_provider.go

```diff
@@ -0,0 +1,45 @@
+package provider
+
+import (
+	"context"
+
+	"github.com/mesg-foundation/core/protobuf/coreapi"
+	"github.com/mesg-foundation/core/utils/workflowparser/yaml"
```

Comment:
Change github.com/mesg-foundation/core/utils/workflowparser/yaml to github.com/mesg-foundation/core/utils/workflowparser

Yaml is good package name for parsing yaml (github.com/go-yaml/yaml), but here shouldbe `workflowparser` or `workflow/parser`. If you want to put emphasize on format, you can create
```go
type Parser interface {
  Parse() (Definition, error)
}

type YAMLParser struct {} // ....
```

Or just create 
```go
workflowparser.ParseFromYAMLFile()
```

Reply:
done